gems/pending/util/system/evm_watchdog.rb

```diff
@@ -35,6 +39,13 @@ def self.check_evm
     end
   end
 
+  def self.check_for_update
+    return unless File.exist?(UPDATE_FILE)
+    packages = File.read(UPDATE_FILE).split(",")
```

Comment:
Even if that is the case, we would just catch the write on the next check right?